src/Component/tests/spec/State/ResponderSpec.php

```diff
@@ -11,15 +11,15 @@
 
 declare(strict_types=1);
 
-namespace spec\Sylius\Component\Resource\State;
+namespace spec\Sylius\Resource\State;
```

Comment:
Missing case for `Assert`, like for processor and provider

```patch
+++ function it_throws_an_exception_when_configured_responder_is_not_a_responder_instance()
```

Reply:
fixed